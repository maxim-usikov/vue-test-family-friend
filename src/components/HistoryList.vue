<template>
  <ul :class="$style.Container">
    <slot />
  </ul>
</template>

<script>
export default {
  name: 'HistoryList',
};
</script>

<style module>
.Container {
  background: rgba(234, 234, 234, 1);
  list-style: none;
  margin: 0;
  padding: 10px;
}
</style>
