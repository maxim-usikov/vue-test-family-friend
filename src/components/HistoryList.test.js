import { shallowMount } from '@vue/test-utils';

import HistoryList from './HistoryList.vue';

describe('HistoryList', () => {
  let $c;

  beforeEach(() => {
    $c = shallowMount(HistoryList);
  });

  it('is a Vue instance', () => {
    expect($c.isVueInstance()).toBeTruthy();
  });

  it('name is `HistoryList`', () => {
    expect($c.name()).toBe('HistoryList');
  });

  it('renders correctly', () => {
    expect($c.element).toMatchSnapshot();
  });
});
