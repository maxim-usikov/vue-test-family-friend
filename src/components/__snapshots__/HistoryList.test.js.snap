// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HistoryList renders correctly 1`] = `
<ul
  class="Container"
/>
`;
